extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -94,6 +94,12 @@ public interface SmallRyeOpenApiConfig {
     @WithDefault("true")
     boolean autoAddTags();
 
+    /**
+     * This will automatically add BadRequest (400 Http status) API response to operation that has an input.
+     */
+    @WithDefault("true")
```

Comment:
Isn't this a breaking change? Do we want it, or should we disable this by default (I don't know if we should consider this PR as a bug fix or enhancement)

Reply:
I think it's a bug, as the JAX-RS default and openapi default is not aligned.